### ISSUE TITLE
3.73 fuse mount read panic

### ISSUE THREAD

**Issue by Ruakij** [seaweedfs/seaweedfs-csi-driver#180]
## Description
As the latest version of seaweed-csi-driver uses 3.73 of [seaweedfs](https://github.com/seaweedfs/seaweedfs) and by default sets `cacheCapacityMB=0`, this issue affects seaweedfs-csi-driver: https://github.com/seaweedfs/seaweedfs/issues/6020

This is mainly informational while the issue persists, as it broke my cluster and i had to search for the error, so others can easily find this.

A hotfix which includes https://github.com/seaweedfs/seaweedfs/pull/6021 might be nice? Not sure if a hotfix is viable, e.g. because the next version is close anyways?

## Symtoms
- Mounting works fine
- When trying to read a file: `Transport endpoint is not connected`
And in `seaweedfs-csi-driver-node`:
```log
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0x1d648c0]

goroutine 98 [running]:
github.com/seaweedfs/seaweedfs/weed/util/chunk_cache.(*TieredChunkCache).GetMaxFilePartSizeInCache(0x10?)
        /github/workspace/weed/util/chunk_cache/chunk_cache.go:51
github.com/seaweedfs/seaweedfs/weed/filer.(*ChunkReadAt).readChunkSliceAt(0xc0008f5680, {0xc000bd4000, 0x10000, 0x10000}, 0xc000c91920, 0xc000c45ef0, 0x0)
        /github/workspace/weed/filer/reader_at.go:202 +0xff
github.com/seaweedfs/seaweedfs/weed/filer.(*ChunkReadAt).doReadAt(0xc0008f5680, {0xc000bd4000, 0x10000, 0x10000}, 0x0)
        /github/workspace/weed/filer/reader_at.go:157 +0x305
github.com/seaweedfs/seaweedfs/weed/filer.(*ChunkReadAt).ReadAtWithTime(0xc000c57140?, {0xc000bd4000?, 0xbd7c43e?, 0x40ea3f?}, 0xc0005a5f80?)
        /github/workspace/weed/filer/reader_at.go:125 +0x119
github.com/seaweedfs/seaweedfs/weed/filer.(*FileChunkSection).readDataAt(0xc000c57140, 0xc000e8d380?, 0x0?, {0xc000bd4000, 0x10000, 0x10000}, 0x0)
        /github/workspace/weed/filer/filechunk_section.go:104 +0xdc
github.com/seaweedfs/seaweedfs/weed/filer.(*ChunkGroup).ReadDataAt(0xc000e8d350, 0xbd7c43e, {0xc000bd4000, 0x10000, 0x10000}, 0x0)
        /github/workspace/weed/filer/filechunk_group.go:63 +0x256
github.com/seaweedfs/seaweedfs/weed/mount.(*FileHandle).readFromChunks(0xc000d98800, {0xc000bd4000, 0x10000, 0x10000}, 0x0)
        /github/workspace/weed/mount/filehandle_read.go:58 +0x53b
github.com/seaweedfs/seaweedfs/weed/mount.readDataByFileHandle({0xc000bd4000, 0x10000, 0x10000}, 0xc000d98800, 0x0)
        /github/workspace/weed/mount/weedfs_file_read.go:81 +0xe5
github.com/seaweedfs/seaweedfs/weed/mount.(*WFS).Read(0x0?, 0xc0008b2fc8?, 0xc0008b3160, {0xc000bd4000, 0x10000, 0x10000})
        /github/workspace/weed/mount/weedfs_file_read.go:49 +0x133
github.com/hanwen/go-fuse/v2/fuse.doRead(0xc0009381a0, 0xc0008b2fc8)
        /go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.5.1/fuse/opcode.go:398 +0x7c
github.com/hanwen/go-fuse/v2/fuse.(*Server).handleRequest(0xc0009381a0, 0xc0008b2fc8)
        /go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.5.1/fuse/server.go:527 +0x2d6
github.com/hanwen/go-fuse/v2/fuse.(*Server).loop(0xc0009381a0, 0x1)
        /go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.5.1/fuse/server.go:500 +0x110
created by github.com/hanwen/go-fuse/v2/fuse.(*Server).readRequest in goroutine 1
        /go/pkg/mod/github.com/hanwen/go-fuse/v2@v2.5.1/fuse/server.go:367 +0x547
```

## Quick-Fix
- Set variable `cacheCapacityMB` to a value >0 (the default for seaweedfs is 1000)

**Comment by chrislusf** [comment]
@Ruakij Thanks! Appreciate the information sharing!

Making a new release v1.2.4 to sync with SeaweedFS 3.74

**Comment by Ruakij** [comment]
@chrislusf This seems to be fixed with 3.76, though i have hit this issue here now https://github.com/seaweedfs/seaweedfs/issues/6110
Not sure why exactly it happens, but it takes some time, couldnt reproduce the exact scenario.
But if that commit fixes the issue, would you mind doing another sync with 3.77 for the csi-driver?
